<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>415</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>438</width>
    <height>415</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Socket Multiplexter v1.1</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="on_pushButton_2_clicked">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>159</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dynamic</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Drone Video Input Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of ports.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP Video</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Drone Telemetry Input Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UdpPort">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>61010</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NumUdpPorts">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TcpPort">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61030</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TelemetruUDPPorts">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>61020</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TcpPort_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61040</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP Telemetri</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="UseTCPVideo">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP Video</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="UseTCPTelemetri">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP Tele</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Static</string>
     </attribute>
     <widget class="QLineEdit" name="SUdpAddress1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>22</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>61030</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>42</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>42</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61031</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>62</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>62</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61012</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>82</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>82</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61013</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>102</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>102</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61014</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>22</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>22</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61020</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress7">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>42</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>42</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61021</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>62</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_8">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>62</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61022</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>82</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_9">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>82</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61023</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpAddress10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>102</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUdpPort_10">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>102</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>61024</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>381</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Addresse / Port      Video                                            Telemetri</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Serial</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>141</width>
        <height>60</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>141</width>
        <height>60</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_4">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>60</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
